CHANGELOG.md

```diff
@@ -25,8 +84,15 @@ Changelog
   * vrf member
 * Move misc READMEs into /docs
 
-0.9.0
------
+## 0.9.0
 
 * First public release, corresponding to Early Field Trial (EFT) of
   Cisco NX-OS 7.0(3)I2(1).
+
+[git-flow]: https://github.com/petervanderdoes/gitflow-avh
+[SimpleCov]: https://github.com/colszowka/simplecov
+
+[v1.1.0]: https://github.com/cisco/cisco-network-node-utils/compare/v1.0.1...v1.1.0
```

Comment:
@glennmatthews This link will not work until we actually create the git tag after collapse into master right?